zipline/finance/blotter.py

```diff
@@ -77,7 +78,8 @@ def __repr__(self):
     def set_date(self, dt):
         self.current_dt = dt
 
-    def order(self, sid, amount, style, order_id=None):
+    def order(self, sid, amount, style, cancel_policy=NeverCancel(),
```

Comment:
Is part of the design that in the same simulation/algorithm, that individual orders can have different cancellation policies?